gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapTypeItem.java

```diff
@@ -0,0 +1,30 @@
+package com.google.android.gnd.ui.home.mapcontainer;
```

Comment:
Please add copyright headers here and throughout.

Reply:
Done.